contracts/TalentLayerEscrow.sol

```diff
@@ -662,7 +662,7 @@ contract TalentLayerEscrow is Initializable, ERC2771RecipientUpgradeable, UUPSUp
         if (transaction.receiverFee != 0) {
             receiverFee = transaction.receiverFee;
             transaction.receiverFee = 0;
-            payable(transaction.receiver).call{value: receiverFee}("");
+            payable(transaction.receiver).transfer(receiverFee);
```

Comment:
@Pfed-prog we used call on purpose, transfer got gas limitation that can causes issues (same for all updates on this commit)

Reply:
Sure, what are those issues?

Reply:
@0xRomain

Reply:
it's an old issue, you can read more about it here: https://consensys.net/diligence/blog/2019/09/stop-using-soliditys-transfer-now/

Reply:
`(bool success, ) = msg.sender.call.value(amount)("");
        require(success, "Transfer failed."); `
would solve warnings